cernopendata/config.py

```diff
@@ -133,6 +137,66 @@
 # Celery
 CELERY_ACCEPT_CONTENT = ["json", "msgpack", "yaml"]
 
+STATS_EVENTS = EVENTS_CONFIG
+
+STATS_AGGREGATIONS = AGGREGATIONS_CONFIG
+
+STATS_QUERIES = {
```

Comment:
I think you miss the `params.index_interval`. Does it make sense to already move it to yearly?

Reply:
Thanks, I'll put it